### PR TITLE
quorum updation

### PR DESCRIPTION
**Pull request by deepakkumardbd**
#505

### CODE REVIEW

File: platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl [hunk 1]
```diff
@@ -54,7 +54,11 @@ spec:
 {% endfor %}
 {% endif %}
     constellation:
+{% if network.config.tm_tls == 'strict' %}
+      url: https://{{ peer.name }}.{{ external_url }}:{{ peer.transaction_manager.ambassador }}/
```
Comment (sownak) [inline platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl:58]: does this work?

File: platforms/quorum/charts/node_constellation/templates/deployment.yaml [hunk 2]
```diff
@@ -169,8 +169,8 @@ spec:
         - "-cx"
         - |-
           #!/usr/bin/env sh
-          sleep 5;
-          
+          apk add curl;
+          # sleep 5;
```
Comment (sownak) [inline platforms/quorum/charts/node_constellation/templates/deployment.yaml:173]: remove commented code

File: examples/supplychain-app/quorum/smartContracts/truffle-config.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ module.exports = {
       timeoutBlocks: 200
     },
     manufacturer: {
-      host: "manufacturer.quo.demo.aws.blockchaincloudpoc.com",     // Localhost (default: none)
+      host: "manufacturerkd.rc.dev.aws.blockchaincloudpoc-develop.com",     // Localhost (default: none)
```
Comment (sownak) [inline examples/supplychain-app/quorum/smartContracts/truffle-config.js:56]: dont check in these changes. You can comment the Web3 though.

File: platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl (hunk 1 above)
Comment (deepakkumardbd) [reply platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl:58]: yes, in netwok.yaml we are specifying http for tm_nodes that goes to othernodes placeholder and I think that's why it is working.

**Pull request merged**